docs/4-roles-expectations.md

```diff
@@ -14,33 +14,49 @@ It should be noted that the Guild as an entity does not directly manage or parti
+- Fully open source, ie. both “source available” and free to fork, modify, redistribute
+- The full focus of the individual (anything less receives a partial weighting, see [6.3 weighting](https://protocol-guild.readthedocs.io/en/latest/6-guidelines-for-regular-operation.html#weighting))
+- Continuous for at least 6 months ahead of inclusion and ongoing
 
-Though Ethereum is a highly technical field, it is clear that the work of non-developers around the core protocol is also crucial and should be eligible. This may include people affiliated with the All Core Devs call, those coordinating ecosystem upgrades, and anyone helping to coevolve the protocol alongside the community.
+Qualifying contributions _must_ target at least one of the following projects/areas:
 
-There may also be contributors to Protocol Guild itself whose work is significant enough to consider for membership. This work may include: 
+1. Ethereum L1 core protocol maintenance and development
 
-- general comms/public presentations to improve awareness or explain the mechanism
-- fundraising from the broader ecosystem (eg. funding programs, individuals, larger entities who depend on Ethereum)
-- research related to the evolution of the Guild itself: smart contract architecture, surveys to members, documentation
-- internal maintenance for the Guild membership: helping with curation processes, onboarding/offboarding members
+- Production client implementations: [Erigon](https://github.com/ledgerwatch/erigon), [Geth](https://github.com/ethereum/go-ethereum), [Hyperledger Besu](https://github.com/hyperledger/besu), [Lighthouse](https://github.com/sigp/lighthouse), [Lodestar](https://github.com/ChainSafe/lodestar), [Nethermind](https://github.com/NethermindEth/nethermind), [Nimbus](https://github.com/status-im/nimbus-eth2), [Prysm](https://github.com/prysmaticlabs/prysm), [Teku](https://github.com/ConsenSys/teku)
+- Client Testing/security/infra which supports these implementations: [ethereum/tests](https://github.com/ethereum/tests) [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests)
+- Coordination related to upgrades and maintenance: [ethereum/pm](https://github.com/ethereum/pm)
+
+2. Research and implementation experiments related to potential protocol changes/refinements
+- [Verkle Tries](https://github.com/gballet/go-verkle) (Verge)
+- [Portal Network](https://github.com/ethereum/portal-network-specs) (Purge)
+- EVM Improvements eg. [Ipsilon](https://github.com/ipsilon)
+
+3. Spec work resulting from the above (should be implementation agnostic, unopinionated)
+- [Execution Specs(EELS)](https://github.com/ethereum/execution-specs)
+- [Consensus Specs](https://github.com/ethereum/consensus-specs)
+
+4. Protocol Guild itself
```

Comment:
This was discussed internally as a very exceptional case and listing it in the core section in the same footing as other projects may raise the idea that we are willing to increase the count of members in this area, something that will meet a strong resistance IMO

Reply:
putting it all in the same place was just for convenience. happy to move it if others feel the same way

Reply:
added language and separated out with this commit https://github.com/protocolguild/docs/pull/47/commits/5847f3fcbb029ee60ebb1c16074daae1028f9054